Source/WebCore/page/Navigation.cpp

```diff
@@ -40,6 +40,28 @@ Navigation::Navigation(ScriptExecutionContext* context, LocalDOMWindow& window)
 {
 }
 
+// https://html.spec.whatwg.org/multipage/nav-history-apis.html#dom-navigation-cangoback
+bool Navigation::canGoBack() const
+{
+    if (hasEntriesAndEventsDisabled())
+        return false;
+    ASSERT(m_currentEntryIndex.has_value());
```

Comment:
This is an std::optional, right? Can't we just assert those without has_value? When you use them in a conditional you don't need has_value either.

Reply:
Good point, fixed.